core/src/main/resources/application.conf

```diff
@@ -255,38 +255,38 @@ backend {
     //        }
     //      }
     //    }
-    }
   }
+}
 
-  services {
-    KeyValue {
-      class = "cromwell.services.KeyValueServiceActor"
-    },
-    MetadataService {
-      class = "cromwell.services.EngineMetadataServiceActor"
-    }
+services {
+  KeyValue {
+    class = "cromwell.services.KeyValueServiceActor"
+  },
+  MetadataService {
+    class = "cromwell.services.EngineMetadataServiceActor"
   }
+}
 
-  database {
-    config = main.hsqldb
+database {
+  config = main.hsqldb
```

Comment:
I'd add a comment saying that the databases listed below aren't **all** active by default, and that you have to pick one to actually use, with this option